client/my-sites/plan-price/index.tsx

```diff
@@ -231,10 +231,9 @@ function FlatPriceDisplay( {
 	className?: string;
 	isSmallestUnit?: boolean;
 } ) {
-	const { symbol: currencySymbol, symbolPosition } = getCurrencyObject(
-		smallerPrice,
-		currencyCode
-	);
+	const { symbol, symbolPosition } = getCurrencyObject( smallerPrice, currencyCode );
+	const currencySymbol = symbol.replace( /^US/, '' );
```

Comment:
Why does this remove the `US` from `US$`? It seems like that defeats the purpose of having it added in the first place (which is to help users from other currencies that use `$` from being confused about what currency they are seeing).

Reply:
I'm going to revert this part of the change here: https://github.com/Automattic/wp-calypso/pull/81228

Remember that `PlanPrice` is not just used on the pricing page.

Reply:
I can wait, though, if you want to revert this whole PR @CodeyGuyDylan (based on the Slack thread).

Reply:
Yes I think I will go ahead and do that